settlement/grpc/grpc.go

```diff
@@ -51,6 +51,16 @@ type Client struct {
 	refreshTime    int
 }
 
+func (c *Client) GetRollapp() (*types.Rollapp, error) {
+	// TODO implement me
+	panic("implement me")
```

Comment:
needs to be fixed for e2e tests

Reply:
i dont think e2e tests are using grpc sl, but implemented just in case..